Userland/Libraries/LibGfx/SkiaBackendContext.cpp

```diff
@@ -41,18 +42,13 @@ class SkiaVulkanBackendContext final : public SkiaBackendContext {
 
     ~SkiaVulkanBackendContext() override { }
 
-    void flush_and_submit() override
+    void flush_and_submit(SkSurface* surface) override
     {
-        m_context->flush();
+        GrFlushInfo const flush_info;
```

Comment:
Do we know that this object has a suitable default constructor? explicitly zero-init (or default init)ing this struct would look less suspicious

```suggestion
        GrFlushInfo const flush_info{};
```

Reply:
yes, using default constructor is suitable here. I added zero-init.